### PR TITLE
change terminology

### PR DESCRIPTION
**Pull request by Joel-Joseph-George**
#608

### CODE REVIEW

File: app/crud/nlp_crud.py [hunk 1]
```diff
@@ -126,19 +126,19 @@ def replace_bulk_tokens(db_, sentence_list, token_translations, src_code, trg_co
     '''Substitute tokens with provided trabslations and get updated drafts, draftMetas
     and add knowledge to translation memory'''
     use_data = kwargs.get("use_data_for_learning",True)
-    source = db_.query(db_models.Language).filter(
+    resource = db_.query(db_models.Language).filter(
```
Comment (shimilgithub) [inline app/crud/nlp_crud.py]: I think in `translation` module, terminology change is not required unless we are referring the `resources` table. @kavitharaju  can you please confirm this?

Comment (Joel-Joseph-George) [reply app/crud/nlp_crud.py]: ohh. Just confirm it and will make the changes as per need

Comment (kavitharaju) [reply app/crud/nlp_crud.py]: Yes, normally `source` in translation module refers to the source of translation as in source ad target. We should not change that to resource

File: app/test/test_agmt_translation2.py
Comment (shimilgithub) [inline app/test/test_agmt_translation2.py:1]: Still you  `resource` and related terms  is used in translation test. it has to be `source` . eg:source_sentences

File: app/test/test_smast_translation2.py [hunk 2]
```diff
@@ -20,7 +20,7 @@
     "targetLanguageCode": "ml"
 }
 
-source_sentences = [
+resource_sentences = [
```
Comment (shimilgithub) [inline app/test/test_smast_translation2.py]: should be `source_sentences`

File: app/test/test_smast_translation2.py [hunk 3]
```diff
@@ -152,13 +152,13 @@ def test_draft_update_negative():
         headers=headers_auth, json=put_data)
     assert resp.json()['error'] == "Requested Content Not Available"
 
-    # upload source and repeat last action
-    put_data_source = {
+    # upload resource and repeat last action
+    put_data_resource = {
```
Comment (shimilgithub) [inline app/test/test_smast_translation2.py]: should be `put_data_source`

**Review by shimilgithub — CHANGES_REQUESTED** [review]
few changes required in ` test_agmt_translation2` and `test_smast_translation2` files as mentioned. Also remove the print statements you have kept in between.

**Review by shimilgithub — APPROVED** [review]

**Pull request merged**